reducers/views/websocket.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {GeneralTypes, UserTypes} from 'mattermost-redux/action_types';
+
+function getInitialState() {
+    return {
+        connected: false,
+        lastConnectAt: 0,
+        lastDisconnectAt: 0,
+    };
+}
+
+export default function(state = getInitialState(), action) {
```

Comment:
Could this be moved into mattermost-redux? It seems like this will be needed on the mobile app as well

Reply:
This is copied from mobile repo so this can definitely be moved to redux repo. I just had a look at the mobile dependency of it and we would have to update mobile repo with more than few changes. 

Can we address that with another PR later? Probably we can add a ticket to scrolling_overhaul to keep a track of it?

Reply:
Yeah, that can be addressed in another PR